parachain/pallets/assets/src/lib.rs

```diff
@@ -0,0 +1,204 @@
+//! # Assets
+//!
+//! The assets pallet provides functionality for handling account balances.
+//!
+//! ## Overview
+//!
+//! This module is used by the ETH and ERC20 pallets to store account balances for bridged assets.
+//!
+//! Each asset is identified by a unique `H160` hash. This is useful for tracking ERC20 tokens which are natively identified by a 20-byte contract address.
+//!
```

Comment:
I didn't think it was the right place for the comparison, since this is supposed to be sterile API reference documentation.

Also I wrote the comparison in the very early stages of the project when I was unsure of whether I was taking the right approach and wanted to motivate to reviewers why I made the choice. But I think we are past that now.

Reply:
Okay, makes sense to me 👍